relm-derive/src/gen/mod.rs

```diff
@@ -36,34 +36,23 @@ use std::collections::{HashMap, HashSet};
 
 use proc_macro2::{Span, TokenStream};
 use quote::{quote, quote_spanned};
-use syn::{
-    Generics,
-    Ident,
-    ImplItem,
-    ImplItemMethod,
-    ItemImpl,
-    Macro,
-    Path,
-    PatType,
-    PathArguments,
-    ReturnType,
-    Signature,
-    TypePath,
-    parse,
-};
-use syn::FnArg::{self, Typed};
 use syn::fold::Fold;
-use syn::ImplItem::{Const, Method, Verbatim};
-use syn::Item::{self, Impl};
 use syn::parse::Result;
 use syn::spanned::Spanned;
-use syn::Type;
 use syn::visit::Visit;
+use syn::FnArg::{self, Typed};
+use syn::ImplItem::{Const, Method, Verbatim};
+use syn::Item::{self, Impl};
+use syn::Type;
+use syn::{
+    parse, Generics, Ident, ImplItem, ImplItemMethod, ItemImpl, Macro, PatType, Path,
```

Comment:
I prefer to have one per line as to avoid useless git conflicts, so please format it like it was before.